src/iptvsimple/utilities/StreamUtils.cpp

```diff
@@ -256,3 +286,24 @@ bool StreamUtils::SupportsFFmpegReconnect(const StreamType& streamType, const ip
          channel.GetProperty(PVR_STREAM_PROPERTY_INPUTSTREAMCLASS) == PVR_STREAM_PROPERTY_VALUE_INPUTSTREAMFFMPEG ||
          channel.GetProperty(PVR_STREAM_PROPERTY_INPUTSTREAMADDON) == PVR_STREAM_PROPERTY_VALUE_INPUTSTREAMFFMPEG;
 }
+
+std::string StreamUtils::UrlEncodeProtocolOptions(const std::string& protocolOptions)
```

Comment:
Maybe call the function `GetUrlEncodedProtocolOptions`?

Might be nice to all have a `HasProtocolOptions` function.